guide/popular-topics/canvas.md

```diff
@@ -56,7 +56,7 @@ client.login('your-token-goes-here');
 ```
 
 ::: warning
-Remember to register the slash commands before continuing on with this section of the guide. You can view how to do that [here](/interactions/registering-slash-commands.md).
+Remember to register the slash commands before continuing on with this section of the guide. You can view how to do that [here](/interactions/slash-commands.md).
```

Comment:
Same as on the other page, this was split for the mentioned reason and should be kept as is.

Reply:
👍🏻 i will split my "single page" into three files and revert this back